rotkehlchen/tests/api/test_data_import.py

```diff
@@ -453,6 +455,42 @@ def test_data_import_binance_history(rotkehlchen_api_server):
     assert_binance_import_results(rotki)
 
 
+def test_data_import_rotki_generic_trades(rotkehlchen_api_server):
+    """Test that data import works for Rotki generic trades import csv file."""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    dir_path = Path(__file__).resolve().parent.parent
+    filepath = dir_path / 'data' / 'rotki_generic_trades.csv'
+
+    json_data = {'source': 'rotki trades', 'file': str(filepath)}
+    response = requests.put(
+        api_url_for(
+            rotkehlchen_api_server,
+            'dataimportresource',
+        ), json=json_data,
+    )
+    result = assert_proper_response_with_result(response)
+    assert result is True
+    assert_rotki_generic_trades_import_results(rotki)
+
+
+def test_data_import_rotki_generic_events(rotkehlchen_api_server):
+    """Test that data import works for Rotki generic events import csv file."""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    dir_path = Path(__file__).resolve().parent.parent
+    filepath = dir_path / 'data' / 'rotki_generic_events.csv'
+
+    json_data = {'source': 'rotki events', 'file': str(filepath)}
+    response = requests.put(
+        api_url_for(
+            rotkehlchen_api_server,
+            'dataimportresource',
+        ), json=json_data,
+    )
+    result = assert_proper_response_with_result(response)
+    assert result is True
```

Comment:
Why split it in two lines?

```suggestion
    assert assert_proper_response_with_result(response) is True
```